Aspire Budgeting/Views/CategoryDetailsView.swift

```diff
@@ -56,7 +56,43 @@ extension CategoryDetailsView {
         .shadow(radius: 10)
         .edgesIgnoringSafeArea(/*@START_MENU_TOKEN@*/.all/*@END_MENU_TOKEN@*/)
 
-      totals
+      VStack {
+        totals
+
+        Divider()
+          .padding([.top, .horizontal])
+
+        Text("Budget per category")
+          .font(.karlaBold(size: 15))
+          .foregroundColor(.secondaryTextColor)
+
+        ScrollView {
+          ForEach(cardDetails.categories, id: \.self) { category in
+            VStack(alignment: .leading) {
+              HStack {
+                Text(category.categoryName)
+                  .font(.karlaBold(size: 14))
+                  .foregroundColor(.primaryTextColor)
+                Spacer()
+                AspireNumberView(number: category.available)
+              }
+              HStack {
+                Text("\(category.spent.stringValue) spent • \(category.budgeted.stringValue) budgeted")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/16f63f2343d9e7b69eb70c1fc9896bed3fcc7422/Aspire Budgeting/Views/CategoryDetailsView.swift#L80"></span>Line should be 100 characters or less: currently 103 characters<br />
<code>line_length</code> <code>CategoryDetailsView.swift:80</code></td>
    </tr>
  </tbody>
</table>